src/lib/yang/yang.lua

```diff
@@ -53,8 +53,10 @@ function load_configuration(filename, opts)
    end
    local function err(msg, ...) error(err_msg(msg, ...)) end
    local function log(msg, ...)
-      io.stderr:write(err_msg(msg, ...)..'\n')
-      io.stderr:flush()
+      if opts.verbose then
```

Comment:
I think we need to change the default above at line 34 to `verbose={default=true}`, to keep the current behavior for existing callers.